web_widget_heatmap/static/src/css/heatmap_widget.css

```diff
@@ -0,0 +1,12 @@
+/* Copyright 2019 Dinar Gabbasov <https://it-projects.info/team/GabbasovDinar>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+.heatmap_widget {
+	width: 100%;
```

Comment:
Can you use spaces here instead of tabs

Reply:
Thats 

> I like this widget.
> The loading problem can also be solved by making the widget work for Serialized fields. The programmer can make a computed and stored Serialized field which stores the records as JSON. This way every time the partner record opens, not all the sale orders are fetched instead just a text field essentially.

Thats the perfect solution, with a depends in the intresting fields